packages/kubernetes-1.24/kubernetes-tls-crt

```diff
@@ -0,0 +1,3 @@
+{{~#if settings.kubernetes.tls-certificate~}}
+{{base64_decode settings.kubernetes.tls-certificate}}
+{{~/if~}}
```

Comment:
non-blocking: I think the conditional is somewhat redundant since this file will not be rendered unless the setting got set. Same for tls-private-crt

Reply:
Ah nevermind, the `configuration-file` metadata applies for the entire `settings.kubernetes`